src/components/PassportPage.js

```diff
@@ -1,17 +1,62 @@
 import React from 'react';
-import axios from 'axios';
+import { getUserData } from '../actions/loginActions';
 
 class PassportPage extends React.Component {
+  state = {
+    username: '',
+    avatarUrl: '',
+    githubUrl: ''
+  }
+  
   componentDidMount() {
-    axios.get('/api/user').then(res => {
-      console.log(res);
-      console.log(res.data);
-    });
+    getUserData().then(
+      (res) => {
+        const { avatarUrl, githubUrl, username } = res;
+        this.setState({ username, avatarUrl, githubUrl });
+      },
+      (err) => console.log // do something with error? display in UI?
+    );
+  }
+  
+  hanleChange = (e) => {
+    this.setState({ username: e.target.value });
+  }
+  
+  handleSubmit = (e) => {
+    e.preventDefault();
+    // here, we need to do 3 things:
```

Comment:
We probably would want to perform all of this logic on the server so we would just call some server endpoint here, like `/api/user-credentials`.

Otherwise this all looks good to me.